### PR TITLE
Create /api/hello endpoint & resolve the err

### PR DESCRIPTION
**Pull request by juhan26**
Screenshot 2024-01-28 at 11 47 33

Screenshot 2024-01-28 at 11 45 52

### CODE REVIEW

File: src/backend/index.js [hunk 1]
```diff
@@ -1,37 +1,11 @@
...
 
-// setup static content from vite build output dir in dist/ui if NODE_ENV is in development mode
-let STATIC_PATH = "../../dist/ui";
-if (process.env.NODE_ENV !== "production") {
-  STATIC_PATH = path.join(__dirname, "../../dist/ui");
-} else {
-  STATIC_PATH = path.join(__dirname, "ui");
-}
-
-app.use(express.static(STATIC_PATH));
```
Comment (pahrizal) [inline src/backend/index.js]: this part should be included in backend entrypoint to be able serving a static content. please add it back

File: src/backend/index.js [hunk 2]
```diff
@@ -1,37 +1,11 @@
...
-app.get("/api/hello", (req, res) => {
-  res.send("Hello World! how are you");
-});
-
-// serve all requests to index.html in production
-if (process.env.NODE_ENV === "production") {
-  app.get("*", (req, res) => {
-    res.sendFile(path.join(STATIC_PATH, "index.html"));
-  });
-}
```
Comment (pahrizal) [inline src/backend/index.js]: this block of lines also needed when we deploy the app in production. please add it back

File: src/backend/index.js [hunk 3]
```diff
@@ -1,37 +1,11 @@
...
+app.use('/api/hello', helloRoute);
 
-// start server
-const server = http.createServer(app);
-server.listen(API_PORT, () => {
-  console.log(`Server listening on port ${API_PORT}`);
+const PORT = 3000;
+app.listen(PORT, () => {
+  console.log(`Server berjalan di http://localhost:${PORT}`);
 });
```
Comment (pahrizal) [inline src/backend/index.js]: use this  `http` server wrapper to start the express app. cause you will need it when you have to implement websocket conection in the future

**Review by pahrizal — CHANGES_REQUESTED** [review]
please add back some codes that has been removed

File: src/backend/index.js (hunk 3 above)
Comment (juhan26) [reply src/backend/index.js]: alright sir

**Comment by juhan26** [comment]
done sir

**Review by pahrizal — APPROVED** [review]
Great!

**Pull request merged**